host/labuser

```diff
@@ -1 +1 @@
-labuser ALL=(ALL) NOPASSWD:ALL
\ No newline at end of file
+labuser ALL=(ALL) NOPASSWD:ALL ROLE=sysadm_r TYPE=sysadm_t ALL
```

Comment:
I manually modified my `host01` container by adding `ROLE=sysadm_r TYPE=sysadm_t ALL` to the proper supers file and then tried to run `ansible -i inventory all -m ping` from the `controller` as root:

```
[root@controller ~]# ansible -i inventory all -m ping
host01 | FAILED! => {
    "msg": "Missing sudo password"
}
host02 | SUCCESS => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "ping": "pong"
}
```

Reply:
please try removing the last " ALL"